config/configs/default_ad.exs

```diff
@@ -0,0 +1,7 @@
+use Mix.Config
+
+config :code_fund, CodeFundWeb.Endpoint,
```

Comment:
if we set this config like so
``` 
config :code_fund, AdService.Advertisement, 
```
instead of on Endpoint, you can pull the whole config out into a list  in the controller.